Use layer number as example for COMBO_ONLY_FROM_LAYER (#18072)
qmk/qmk_firmware: docs/feature_combo.md

If you, for example, use multiple base layers for different key layouts, one for QWERTY, and another one for Colemak, you might want your combos to work from the same key positions on all layers. Defining the same combos again for another layout is redundant and takes more memory. The solution is to just check the keycodes from one layer.

> With `#define COMBO_ONLY_FROM_LAYER 0` the combos' keys are always checked from layer `0` even though the active layer would be `1`.

Reviewer: ```suggestion
With `#define COMBO_ONLY_FROM_LAYER 0` in config.h, the combos' keys are always checked from layer `0`, even if other layers are active.
```
